sound/soc/sof/core.c

```diff
@@ -261,6 +262,9 @@ static int sof_probe_continue(struct snd_sof_dev *sdev)
 	if (!sof_ops(sdev)->runtime_suspend || !sof_ops(sdev)->runtime_resume)
 		pm_runtime_get_noresume(sdev->dev);
 
+	/* register SOF client devices now that FW is booted */
+	sof_client_register_clients(sdev);
```

Comment:
do I understand it correctly, that you haven't enabledcalling `.remove()` yet?

Reply:
yes, done